sos/report/plugins/containerd.py

```diff
@@ -0,0 +1,33 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import (Plugin, PluginOpt, RedHatPlugin,
+                                UbuntuPlugin, CosPlugin)
+
+
+class Containerd(Plugin, RedHatPlugin, UbuntuPlugin, CosPlugin):
+
+    short_desc = 'Containerd containers'
+    plugin_name = 'containerd'
+    profiles = ('container',)
+
```

Comment:
There should be some enablement trigger here. Either a `packages`, `files`, or `commands` tuple will suffice. If the package name differs between distros, that's still fine to have a single plugin - just put all the possible package names in the tuple and sos will figure it out.